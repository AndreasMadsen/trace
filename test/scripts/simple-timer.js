'use strict';

//
// test global method (setTimeout)
//

setTimeout(function callback() {
  throw new Error('trace');
}, 0);
