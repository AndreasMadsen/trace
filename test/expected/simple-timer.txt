/scripts/simple-timer.js:8
  throw new Error('trace');
  ^

Error: trace
    at Timeout.callback [as _onTimeout] (/scripts/simple-timer.js:8:9)
    at Object.<anonymous> (/scripts/simple-timer.js:7:1)
